src/service/service.py

```
@@ -117,7 +125,9 @@ async def info() -> ServiceMetadata:
     )
 
 
-async def _handle_input(user_input: UserInput, agent: AgentGraph) -> tuple[dict[str, Any], UUID]:
+async def _handle_input(
+    user_input: UserInput, agent: AgentGraph
```

Ruff (0.14.11)

Unused function argument: `agent`

(ARG001)